tests/src/test/java/alluxio/client/fs/FileOutStreamAsyncWriteIntegrationTest.java

```diff
@@ -69,41 +117,45 @@ public void asyncWrite() throws Exception {
     os.close();
 
     CommonUtils.sleepMs(1);
-    checkPersistStateAndWaitForPersist(filePath, 2);
+    IntegrationTestUtils
+        .checkPersistStateAndWaitForPersist(sLocalAlluxioClusterResource, mFileSystem, filePath, 2);
   }
 
   @Test
   public void asyncWriteWithZeroWaitTime() throws Exception {
     AlluxioURI filePath = new AlluxioURI(PathUtils.uniqPath());
-    createTwoBytesFile(filePath, 0);
+    createTwoBytesFile(mFileSystem, filePath, 0);
 
     CommonUtils.sleepMs(1);
-    checkPersistStateAndWaitForPersist(filePath, 2);
+    IntegrationTestUtils
+        .checkPersistStateAndWaitForPersist(sLocalAlluxioClusterResource, mFileSystem, filePath, 2);
   }
 
   @Test
-  @LocalAlluxioClusterResource.Config(
-      confParams = {PropertyKey.Name.USER_FILE_PERSIST_ON_RENAME, "true"})
   public void asyncWriteRenameWithNoAutoPersist() throws Exception {
-    AlluxioURI srcPath = new AlluxioURI(PathUtils.uniqPath());
-    AlluxioURI dstPath = new AlluxioURI(PathUtils.uniqPath());
-    createTwoBytesFile(srcPath, Constants.NO_AUTO_PERSIST);
-
-    CommonUtils.sleepMs(1);
-    // check the file is completed but not persisted
-    URIStatus srcStatus = mFileSystem.getStatus(srcPath);
-    assertEquals(PersistenceState.TO_BE_PERSISTED.toString(), srcStatus.getPersistenceState());
-    Assert.assertTrue(srcStatus.isCompleted());
-
-    mFileSystem.rename(srcPath, dstPath);
-    CommonUtils.sleepMs(1);
-    checkPersistStateAndWaitForPersist(dstPath, 2);
+    ServerConfiguration.set(PropertyKey.USER_FILE_PERSIST_ON_RENAME, "true");
+    try (FileSystem fs = sLocalAlluxioClusterResource.get().getClient()) {
```

Comment:
why can't we use `mFileSystem`?

Reply:
We could, but all files created here need to be created with the `PERSIST_ON_RENAME` option, so it's easier to set it within `ServerConfiguration` and then create a client based on it so that all creates default to using it.

Also, mainly because the test uses the `createTwoByteFile` method which didn't expose the options previously. So, I just created a new client with the default option instead.